data.md

```diff
@@ -94,13 +93,21 @@ All data is also available on [our GitHub repository for analysis](https://githu
             <span class="visually-hidden">Expand section for {{ file.name }}</span>
             <img id="icon-{{ date | slugify }}-{{ file.name | slugify }}" src="{{ '/assets/icons/triangle-right.svg' | relative_url }}" alt="Expand section for {{ file.name }}" width="16" height="16">
           </button>
-          <span class="file-title">{{ file.name }}</span>
+          {% assign file-descs = site.data.analysis-files | where: 'file', file.name %}
```

Comment:
perhaps we can import all file-descs at the top (line 36) and only filter based on file.name in this loop